src/util.rs

```diff
@@ -106,11 +107,10 @@ impl DebugVisualizations {
         use std::fs::File;
         use std::io::Write;
 
-        // make the directory, clearing it if needed
-        if path.exists() {
-            std::fs::remove_dir_all(&path).unwrap();
+        // make the directory if it doesn't exist
+        if !path.exists() {
+            std::fs::create_dir_all(&path).unwrap();
```

Comment:
we're retuning an io::Result here, can we do `std::fs::create_dir_all(&path)?;` instead?

Reply:
good idea